### PR TITLE
Class Name Specificity

### PR DESCRIPTION
**Pull request by luke-xlab**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,16 +7,16 @@
     <script type="text/javascript" src="app.js"></script>
   </head>
   <body>
-    <div class="fixed__menu">
-      <div class="menu__top">
+    <div class="fixed-menu">
+      <div class="menu--top">
```
Comment (hunterbrowning) [inline index.html]: Lets make these more like `fixed-menu--top` its always a tricky balance of when to use the parents full name or not but in a case like this with a work like "menu" being super vague, we want to.

File: index.html [hunk 2]
```diff
@@ -44,54 +44,54 @@
         </p>
       </div>
       <br>
-      <div class="homepage__circle">
+      <div class="background-video__scroll-circle">
```
Comment (hunterbrowning) [inline index.html]: Whenever possible, lets use things that describe what something is, not its geometry. so in this case, `background-video__scroll-btn` might be more applicable.

File: index.html [hunk 3]
```diff
@@ -44,54 +44,54 @@
...
       </div>
     </div>
     <div class="scroll-pages">
-      <div class="content__header">
+      <div class="navbar--header">
```
Comment (hunterbrowning) [inline index.html]: "header" isn't really a permutation of navbar. Its a subelement. So use __ not --

File: index.html [hunk 4]
```diff
@@ -44,54 +44,54 @@
...
         <p>
           A new <strong>Happening Now and</strong> is ready for you!
         </p>
       </div>
         <section class="content-blocks">
-          <div id="block1" class="content-div-block" style="background-color:white">
+          <div id="block1" class="image-block">
```
Comment (hunterbrowning) [inline index.html]: You might want something more descriptive. Like `project-image-tile`. "image-block" is pretty vague.

File: index.html [hunk 5]
```diff
@@ -44,54 +44,54 @@
...
-          <div id="block7" class="content-div-block" style="background-color:white">
+          <div id="block7" class="image-block">
           </div>
-          <div id="block8" class="content-div-block" style="background-color:white">
+          <div id="block8" class="image-block">
           </div>
         </section>
-        <div class="content__brain">
-          <p id="content__brain-think">
+        <div class="brain--header">
```
Comment (hunterbrowning) [inline index.html]: Again, "header" is not really a permutation. __ not --

File: index.html [hunk 6]
```diff
@@ -44,54 +44,54 @@
...
+          <div class="headset-sound">
           </div>
-          <div class="content__brain-headset">
-            <div class="content__brain-brain">
-                <div class="brain__descriptions">
-                  <div class="top-left__brain" id="brain-layout">
+          <div class="brain-headset">
+            <div class="brain-image">
+                <div class="brain-descriptions">
+                  <div class="top--left-brain" id="brain-image-layout">
```
Comment (hunterbrowning) [inline index.html]: Can you make these make more sense? You typically want to permutation after the --. So "brain" needs to be in front. Ex: `brain--top-left`

File: index.html [hunk 7]
```diff
@@ -103,43 +103,43 @@
             to change the rules of the game. That gives us energy
             and keeps us sharp and critical. And our customers.
           </p>
-          <div class="content__brain-find-more-button">
+          <div class="brain-find-more-button">
```
Comment (hunterbrowning) [inline index.html]: Its fine to abbreviate "button" to 'btn" which is nice for long names like this. If you do it, you should do it everywhere.

File: index.html [hunk 8]
```diff
@@ -103,43 +103,43 @@
...
             <p>
               Find out more about Blink
             </p>
           </div>
         </div>
-        <div class="content__product">
-          <p id="title">
+        <div class="product--header">
+          <p id="product--header-title">
```
Comment (hunterbrowning) [inline index.html]: header is not a permutation

File: index.html [hunk 9]
```diff
@@ -150,21 +150,21 @@ <h4 class="box-content__headers">Blink in jury awards Meet Magento</h4>
                     awards presented. Register now for the Magento
                     event of the Netherlands.
                   </p>
-                  <div class="top-left__profile-image">
+                  <div class="top--left-profile-image">
```
Comment (hunterbrowning) [inline index.html]: Can this follow the same naming pattern as the brain stuff?

File: index.html [hunk 10]
```diff
@@ -150,21 +150,21 @@ <h4 class="box-content__headers">Blink in jury awards Meet Magento</h4>
...
                   </div>
-                  <div class="top-left__profile__description">
-                    <p id="profile__name" class="box-content__para">
+                  <div class="top--left-profile-description">
```
Comment (hunterbrowning) [inline index.html]: Can this follow the same naming pattern as the brain stuff

File: index.html [hunk 11]
```diff
@@ -150,21 +150,21 @@ <h4 class="box-content__headers">Blink in jury awards Meet Magento</h4>
...
+                    <p id="profile-name" class="box-para">
                       <strong style="color: black">Albert Einstein</strong>
                       <br>
                       Absolute Genius
                     </p>
                   </div>
               </div>
-              <div class="block-top-right">
-                <div class="top-image-right">
+              <div class="block--top-right">
```
Comment (hunterbrowning) [inline index.html]: "box" seems pretty vague

File: index.html [hunk 12]
```diff
@@ -173,19 +173,19 @@ <h4 class="box-content__headers">'People first' in your marketing and sales?</h4
...
                     <strong style="color: black">Elon Musk</strong>
                     <br>
                     Innovator
                   </p>
                 </div>
               </div>
-              <div class="block__middle-left">
-                <h4 class="box-content__headers">Common ambition</h4>
-                <p class="box-content__para">
+              <div class="block--middle-left">
```
Comment (hunterbrowning) [inline index.html]: Good use of permutations!

**Comment by hunterbrowning** [comment]
A lot of my comments relate to issues that are all throughout the code. So review it all with those comments in mind. I am not going to leave 30 comments on the same issue, its just annoying with email notifications.

File: index.html (hunk 1 above)
Comment (luke-xlab) [inline index.html]: fixed

File: index.html (hunk 2 above)
Comment (luke-xlab) [inline index.html]: fixed

File: index.html (hunk 3 above)
Comment (luke-xlab) [inline index.html]: fixed in all instances

File: index.html (hunk 4 above)
Comment (luke-xlab) [inline index.html]: fixed

File: index.html (hunk 6 above)
Comment (luke-xlab) [inline index.html]: fixed

File: index.html (hunk 7 above)
Comment (luke-xlab) [inline index.html]: fixed

File: index.html (hunk 9 above)
Comment (luke-xlab) [inline index.html]: fixed

File: index.html (hunk 10 above)
Comment (luke-xlab) [inline index.html]: fixed

File: index.html (hunk 11 above)
Comment (luke-xlab) [inline index.html]: fixed

File: index.html [hunk 13]
```diff
@@ -31,7 +31,7 @@
           </font>
         </h1>
       </div>
-      <div class="homepage__text">
+      <div class="background-video-text">
```
Comment (hunterbrowning) [inline index.html]: I would match this to the above syntax and make it `background-video__text`

File: index.html [hunk 14]
```diff
@@ -44,54 +44,54 @@
...
-          <div id="block7" class="content-div-block" style="background-color:white">
+          <div id="block7" class="project-image-tile">
           </div>
-          <div id="block8" class="content-div-block" style="background-color:white">
+          <div id="block8" class="project-image-tile">
           </div>
         </section>
-        <div class="content__brain">
-          <p id="content__brain-think">
+        <div class="brain__header">
```
Comment (hunterbrowning) [inline index.html]: If it's not a child, your don't need the `__` got with with brain-header. Or, if you have a reason to make a `brain` parent, then your syntax would be correct.

File: index.html [hunk 15]
```diff
@@ -44,54 +44,54 @@
...
+          <p class="brain-think">
```
Comment (hunterbrowning) [inline index.html]: "think" doesn't give me much information without knowing the site. Maybe shoot for something that describes that implementation of text.

File: index.html [hunk 16]
```diff
@@ -44,54 +44,54 @@
...
+          <div class="headset-sound">
           </div>
-          <div class="content__brain-headset">
-            <div class="content__brain-brain">
-                <div class="brain__descriptions">
-                  <div class="top-left__brain" id="brain-layout">
+          <div class="brain-headset">
+            <div class="brain-image">
+                <div class="brain-descriptions">
+                  <div class="brain--top-left" id="brain-image-layout">
```
Comment (hunterbrowning) [inline index.html]: Great job! These cleaned up nicely

File: index.html [hunk 17]
```diff
@@ -103,43 +103,43 @@
...
             </p>
           </div>
         </div>
-        <div class="content__happening">
-          <h1 class="content__happening-header"><strong>NOW AND HAPPENING</strong></h1>
-              <div class="block-top-left">
-                  <div class="top-left__image-left">
+        <div class="happening-main">
+          <h1 class="happening__header"><strong>NOW AND HAPPENING</strong></h1>
+              <div class="block--top-left">
```
Comment (hunterbrowning) [inline index.html]: "block" is probably too vague

File: index.html [hunk 18]
```diff
@@ -103,43 +103,43 @@
...
+                  <div class="image--top-left">
```
Comment (hunterbrowning) [inline index.html]: Same with "image". They either need some prefixing context providing word or themselves need to become more specific.

**Pull request merged**